packages/cli/cmd/application/main.go

```diff
@@ -116,13 +141,28 @@ func buildRootCmd() *cobra.Command {
 	cmd.SetUsageTemplate(template.RootUsage)
 
 	cmd.PersistentFlags().BoolVarP(&logging.Verbose, cli.VerboseFlag, cli.VerboseFlagShort, false, cli.VerboseFlagDescription)
-	cmd.PersistentFlags().StringVar(&vars.format, cli.FormatFlag, cli.FormatFlagDefault, cli.FormatFlagDescription)
+	cmd.PersistentFlags().StringVar(&formatVars.format, cli.FormatFlag, "", cli.FormatFlagDescription)
 	cmd.Flags().StringVar(&vars.docPath, "docs", "", "generate markdown documenting the CLI to the specified path")
 	cmd.Flag("docs").Hidden = true
 
 	return cmd
 }
 
+func setFormatter(opts *formatOpts) string {
+	configClient := opts.configClient
+	formatVars := opts.formatVars
+	if formatVars.format == "" {
+		// read the default format from config
+		defaultFormat, err := configClient.GetFormat()
+		formatVars.format = defaultFormat
```

Comment:
We shouldn't set this value if there was an error with the command. Please move it to the else part of the if below. 
It also looks like we might need to set a default value so we could set that the existing default.